tests/api/controllers/robot_window_html/robot_window_html.c

```diff
@@ -23,9 +23,9 @@ int main(int argc, char **argv) {
   }
 
   while (!received) {  // receive message sent by Webots.
-    const char *test_message = (char *)wb_robot_wwi_receive_text();
     wb_robot_step(TIME_STEP);
-    if (test_message) {
+    const char *test_message;
+    while ((test_message = wb_robot_wwi_receive_text())) {
```

Comment:
```suggestion
    while (test_message = wb_robot_wwi_receive_text()) {
```

Reply:
The parenthesis are here to fix a clang warning